README.rst

```diff
@@ -137,9 +137,13 @@ to merge paths from the config file and from the argument:
 
     $ php php-cs-fixer.phar fix --path-mode=intersection /path/to/dir
 
-The ``--format`` option for the output format. Supported formats are ``txt`` (default one), ``json``, ``xml`` and ``junit``.
+The ``--format`` option for the output format. Supported formats are ``txt`` (default one), ``json``, ``xml``, ``checkstyle`` and ``junit``.
+
+NOTE: the output for the following formats are generated in accordance with XML schemas
+
+* ``junit`` follows the [JUnit xml schema from Jenkins](doc/junit-10.xsd)
+* ``checkstyle`` follows the common ["checkstyle" xml schema](doc/checkstyle.xsd)
```

Comment:
these links don't seem to work, maybe the relative path should be done in another way?

Reply:
don't we have all links with root `/` ?

Reply:
as far as I can see in the README.rst proposed here we don't use it expect here

Reply:
https://raw.githubusercontent.com/FriendsOfPHP/PHP-CS-Fixer/2.7/doc/COOKBOOK-FIXERS.md
grep for `(/`

Reply:
@K-Phoen can you prefix the path with `/` as @keradus suggests to see if the links will work than?

Reply:
Thanks for updating, the links are still rendered as plain text. Maybe this is because the file is `rst` and not `md`?

Reply:
Fixed :)

You're right, I was writing markdown instead of reStructuredText.

Reply:
looking good now, thanks man :+1: